client.go

```diff
@@ -33,6 +34,17 @@ type Client struct {
 	debugLogFunc LogFunc
 }
 
+// ExportedFuncWithBadComent is a bad comment
+func ExportedFuncWithBadComment() {
+	fmt.Printf("asdfasdf")
+}
+
+func NotCommentedExported() int {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NotCommentedExported should have comment or be unexported (golint)